lib/trailblazer/operation/public_call.rb

```diff
@@ -36,8 +43,16 @@ def call_with_public_interface(*args)
     end
 
     # This interface is used for all nested OPs (and the outer-most, too).
+    #
+    # @param [Array] args - Contains [ctx, flow_options]
+    # @param [Hash]  circuit_options - Options to configure activity circuit
+    #
+    # @return [signal, [ctx, flow_options]]
+    #
+    # @private
     def call_with_circuit_interface(args, circuit_options)
-      strategy_call(args, circuit_options) # FastTrack#call
+      ctx = options_for_public_call(args[0], flow_options(args[1]))
```

Comment:
Should we add `ctx, flow_options = args`?

Reply:
Yep, by having `original_flow_options` as method name, we can now define these local variables.
I added `def call_with_circuit_interface((options, flow_options), circuit_options)` instead. `ctx` will be an actual `Trailblazer::Context` object below.